css/aiosp_admin.css

```diff
@@ -132,3 +132,8 @@ li#wp-admin-bar-aioseop-pro-upgrade a.ab-item {
 	color: #d54e21;
 	font-size: 105%;
 }
+
+/* ensure the chosen auto-complete is visible */
+#aiosp_sitemap_excl_terms_wrapper .aioseop_option_div {
+	overflow-y: visible;
+} 
```

Comment:
Might be good measure to add a new line at end of file with this PR

Reply:
done